### PR TITLE
Add Bidirectional annotation

### PR DESCRIPTION
**Pull request by victorg1991**
There were no support for adding bidirectional models using annotations

This PR add this bidirectional annotation

### CODE REVIEW

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/converter/ContactPointConverter.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	 * Converts a {@link ContactPointModel} to a {@link ContactPoint}.
+	 *
+	 * @param  contactPointModel the internal contact point model
+	 * @return a {@link ContactPoint}
+	 * @review
+	 */
+	public static ContactPoint toContactPoint(
+		ContactPointModel contactPointModel) {
+
+		return new ContactPoint() {
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/converter/ContactPointConverter.java]: https://schema.org/ContactPoint ¯\_(ツ)_/¯

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/dao/ContactPointModelService.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+	/**
+	 * Returns the contact point that matches the specified ID, if that
+	 * contact point exists; returns {@code Optional#empty()} otherwise.
+	 *
+	 * @param  id the contact point's ID
+	 * @return the he contact point, if present; {@code Optional#empty()}
+	 *         otherwise
+	 * @review
+	 */
+	public Optional<ContactPointModel> get(long id) {
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/dao/ContactPointModelService.java]: I saw it [here](https://github.com/liferay/com-liferay-apio-architect/blob/master/apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/dao/BlogPostingCommentModelService.java#L122)

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/router/ContactPointActionRouter.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+ */
+@Component(
+	immediate = true,
+	service = {ActionRouter.class, ContactPointActionRouter.class}
+)
+public class ContactPointActionRouter implements ActionRouter<ContactPoint> {
+
+	@Retrieve
+	public ContactPoint retrieve(@Id long id) {
+		Optional<ContactPointModel> optional = _contactPointModelService.get(
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/router/ContactPointActionRouter.java]: I saw it here [here](https://github.com/liferay/com-liferay-apio-architect/blob/master/apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/router/BlogPostingCommentActionRouter.java#L107)

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/resource/ContactPointNestedCollectionResource.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Víctor Galán
+ */
+@Component(service = NestedCollectionResource.class)
+public class ContactPointNestedCollectionResource
+	implements NestedCollectionResource
+		<ContactPoint, Long,
+		 ContactPointNestedCollectionResource.ContactPointIdentifier, Long,
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/resource/ContactPointNestedCollectionResource.java]: that identifier is on the same file, but i can change Person's one

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/type/ContactPoint.java [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+import com.liferay.apio.architect.annotation.Id;
+import com.liferay.apio.architect.annotation.Vocabulary.BidirectionalModel;
+import com.liferay.apio.architect.annotation.Vocabulary.Field;
+import com.liferay.apio.architect.annotation.Vocabulary.Type;
+import com.liferay.apio.architect.identifier.Identifier;
+
+/**
+ * @author Víctor Galán
+ */
+@Type("Category")
```
Comment (ahdezma) [inline apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/type/ContactPoint.java]: If the interface is a `ContactPoint` (referenced [here](https://github.com/liferay/com-liferay-apio-architect/pull/296/commits/d6c9d61557873271a7b7509fea77b6b2cb478a1f#r221599717)? Why is the type a `Category`?

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/converter/ContactPointConverter.java [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+public class ContactPointConverter {
+
+	/**
+	 * Converts a {@link ContactPointModel} to a {@link ContactPoint}.
+	 *
+	 * @param  contactPointModel the internal contact point model
+	 * @return a {@link ContactPoint}
+	 * @review
+	 */
+	public static ContactPoint toContactPoint(
```
Comment (ahdezma) [inline apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/converter/ContactPointConverter.java]: A unit test for this class could be nice :)

All the other converters [have tests](https://github.com/liferay/com-liferay-apio-architect/tree/master/apio-architect-sample/src/test/java/com/liferay/apio/architect/sample/internal/converter)

Comment (victorg1991) [reply apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/converter/ContactPointConverter.java]: Done :D

File: apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/type/ContactPoint.java (hunk 5 above)
Comment (victorg1991) [reply apio-architect-sample/src/main/java/com/liferay/apio/architect/sample/internal/type/ContactPoint.java]: Ups, fixed

**Comment by victorg1991** [comment]
Comments addressed

**Pull request merged**